### PR TITLE
add leparagliding

### PR DESCRIPTION
**Pull request by looooo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/leparagliding/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+${GFORTRAN} leparagliding.f
```
Comment (scopatz) [inline recipes/leparagliding/build.sh:1]: I think this should be using `$FC`

Comment (looooo) [reply recipes/leparagliding/build.sh:1]: does it work for windows too?

Comment (looooo) [reply recipes/leparagliding/build.sh]: @scopatz  I think no (at least there is no such variable in the build). So I guess this should be ready.

Comment (scopatz) [reply recipes/leparagliding/build.sh]: It should.  It is in the activation scripts for the compiler

Comment (looooo) [reply recipes/leparagliding/build.sh]: I tried this command for windows:
`%FC% leparagliding.f`

but I see this in the logs:
`%SRC_DIR%>leparagliding.f`

so I return to calling `gfortran` directly

Comment (scopatz) [reply recipes/leparagliding/build.sh]: @isuruf - shouldn't `%FC%` work for windows too?

Comment (isuruf) [reply recipes/leparagliding/build.sh]: No, `%FC%` is not set.

Comment (looooo) [reply recipes/leparagliding/build.sh]: so feel free to merge : )

Comment (looooo) [reply recipes/leparagliding/build.sh]: @conda-forge/core

**Pull request merged**